lib/__test__/query.test.js

```diff
@@ -130,4 +187,23 @@ describe('query', () => {
         assert.deepEqual(query.getDenominator(parse('-(2/3)')), parse('3'))
         assert.equal(query.getDenominator(x), null)
     })
+
+    it('getPolyDegree', () => {
+        assert.deepEqual(query.getPolyDegree(parse('2')), parse('0'))
+        assert.deepEqual(query.getPolyDegree(parse('2x')), parse('1'))
+        assert.deepEqual(query.getPolyDegree(parse('2x^2')), parse('2'))
+        assert.deepEqual(query.getPolyDegree(parse('x^2')), parse('2'))
+        assert.deepEqual(query.getPolyDegree(parse('-2x^2')), parse('2'))
+        assert.deepEqual(query.getPolyDegree(parse('-2/3 x^2')), parse('2'))
```

Comment:
This function was only meant to get the degree of polynomial terms and not entire polynomials.